src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        StringBuilder builder = new StringBuilder();
+        String text;
+        int supplyAmount = 0;
+        int buyAmount = 0;
+
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            while ((text = bufferedReader.readLine()) != null) {
+                String[] splitLine = text.split(",");
```

Comment:
The use of magic strings 'supply' and 'buy' directly in the code should be avoided. Consider declaring them as constants with meaningful names to improve code readability and maintainability.